<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKiwoom">
    <property name="title">
     <string>kiwoom</string>
    </property>
    <addaction name="loginAction"/>
    <addaction name="separator"/>
    <addaction name="loginStatusAction"/>
    <addaction name="userInfoAction"/>
    <addaction name="depositInfoAction"/>
    <addaction name="accountBalanceInfoAction"/>
    <addaction name="unContractedInfoAction"/>
    <addaction name="realtimeDataAction"/>
    <addaction name="conditionSearchAction"/>
   </widget>
   <widget class="QMenu" name="menuStockInfo">
    <property name="title">
     <string>StockInfo</string>
    </property>
    <addaction name="foreignerPlatformAction"/>
    <addaction name="financialStaPlatformAction"/>
    <addaction name="listedCorporationAction"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="exitAction"/>
    <addaction name="openAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuHelpPlatform"/>
   </widget>
   <addaction name="menuKiwoom"/>
   <addaction name="menuStockInfo"/>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loginAction">
   <property name="text">
    <string>접속하기</string>
   </property>
  </action>
  <action name="loginStatusAction">
   <property name="text">
    <string>접속상태</string>
   </property>
  </action>
  <action name="userInfoAction">
   <property name="text">
    <string>정보보기</string>
   </property>
  </action>
  <action name="depositInfoAction">
   <property name="text">
    <string>예수금조회</string>
   </property>
  </action>
  <action name="accountBalanceInfoAction">
   <property name="text">
    <string>계좌잔고조회</string>
   </property>
  </action>
  <action name="unContractedInfoAction">
   <property name="text">
    <string>미체결내역조회</string>
   </property>
  </action>
  <action name="foreignerPlatformAction">
   <property name="text">
    <string>외국인 한도 소진율</string>
   </property>
  </action>
  <action name="financialStaPlatformAction">
   <property name="text">
    <string>제무제표</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>newWindow</string>
   </property>
  </action>
  <action name="menuHelpPlatform">
   <property name="text">
    <string>Platform</string>
   </property>
  </action>
  <action name="realtimeDataAction">
   <property name="text">
    <string>실시간데이타</string>
   </property>
  </action>
  <action name="listedCorporationAction">
   <property name="text">
    <string>상장사</string>
   </property>
  </action>
  <action name="conditionSearchAction">
   <property name="text">
    <string>조건검색식</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
